functions/index.js

```diff
@@ -336,9 +338,20 @@ exports.editPageTask = functions.https.onRequest((request, response) => {
         var category = request.body.data.Category;
         var storyPointNumber = request.body.data.StoryPointNumber;
         var sprintNumber = request.body.data.SprintNumber;
+        var previousId = request.body.data.PreviousId;
+        var previousSprintId = "S" + previousId;
```

Comment:
Please use createSprintId method here to create previousSprintId, Otherwise in the condition to send the task to backlog it will fail.

Reply:
okay